library/agent/ServiceConfig.ts

```diff
@@ -126,6 +129,32 @@ export class ServiceConfig {
     return { blocked: false };
   }
 
+  private setOnlyAllowedIPAddresses(ipAddresses: IPList[]) {
+    this.onlyAllowedIPAddresses = undefined;
+
+    if (ipAddresses.length === 0) {
+      return;
+    }
+
+    const ips = ipAddresses.map((source) => source.ips).flat();
```

Comment:
Why do we need to use .flat() isn't it already just one array 🤔 , if not add comment